plugins/woocommerce/includes/admin/settings/views/html-admin-page-shipping-zones.php

```diff
@@ -65,6 +65,11 @@
 
 </table>
 
+<?php
+// phpcs:ignore WooCommerce.Commenting.CommentHooks.MissingHookComment
+do_action( 'woocommerce_shipping_zones_after_zones_table' );
```

Comment:
```suggestion
do_action( 'woocommerce_shipping_zones_after_methods_table' );
```
I'd suggest to change the name as above as this is after the shipping methods table on the shipping zones page.

Reply:
I am also curious about this hook. What would you like to add to this page via the hook? We aim to keep these pages focused on their intentions, so adding a general hook like this could mean the page could become cluttered with other things that are not necessarily relevant to shipping zone methods.

Reply:
Hi @kloon and thanks for your interest in this PR.

The shipping methods list inside each zone (Settings > Shipping > Shipping zones > Inside a zone) already has a data-id argument, which allows for direct css or js targeting, but the same does not happen on the general view in Settings > Shipping > Shipping zones. This PR intends to make this coherent on both views.

As for my use case, I have a plugin which allows for pickup location selection on the frontend. On this plugin settings, the shop owner will select which shipping methods will be eligible for that.

<img width="573" alt="image" src="https://github.com/user-attachments/assets/eaaf0a39-4b0f-45a4-a23b-2b0f9b86d68a" />

Our customers (shop owner) have been giving us feedback that it would be useful to have some king of feedback on the shipping zones settings so that they know which methods are selected on our plugin. This PR allows us to target those LI elements and style it:

<img width="1604" alt="image" src="https://github.com/user-attachments/assets/35a7d952-106f-4a1d-aee8-fcc83c63f044" />

I will take care of removing the phpcs comment and add a proper hook comment. Sorry for that :-D

Reply:
> I'd suggest to change the name as above as this is after the shipping methods table on the shipping zones page.


The methods are listed on each table row, but this is actually the shipping zones table, so I think the hook name should not be changed as proposed.

<img width="413" alt="image" src="https://github.com/user-attachments/assets/8eeab80b-eb31-4c8f-be8b-fc9353e3a8e9" />

Reply:
> I'd suggest to change the name as above as this is after the shipping methods table on the shipping zones page.

The methods are listed on each zone row, but the table itself is a zones table, not a methods table (which resides inside each zone details), so I don't think the proposed change on the hook name should be done.

<img width="524" alt="image" src="https://github.com/user-attachments/assets/b5350c01-4a9d-4a72-8b92-55061a8e4eb3" />

Reply:
phpcs comment removed and hook comment added

https://github.com/woocommerce/woocommerce/pull/51817/commits/40d365dfd23e6d40ebf3622f8e642a52f9cfa1ab